google/cloud/storage/examples/storage_bucket_acl_samples.cc

```diff
@@ -231,20 +231,16 @@ void RunAll(std::vector<std::string> const& argv) {
   namespace examples = ::google::cloud::storage::examples;
   namespace gcs = ::google::cloud::storage;
 
+  examples::CheckEnvironmentVariablesAreSet({
+      "GOOGLE_CLOUD_PROJECT",
+      "GOOGLE_CLOUD_CPP_STORAGE_TEST_HMAC_SERVICE_ACCOUNT",
+  });
   auto const project_id =
       google::cloud::internal::GetEnv("GOOGLE_CLOUD_PROJECT").value_or("");
```

Comment:
Q.  Should we still use `value_or()` after `CheckEnvironmentVariablesAreSet()`?

Reply:
Good question, I think not.  I made the changes in this PR, but will send a separate PR to fix `bigtable/examples/`.